### PR TITLE
Update Component,Environment example to working version

### PR DESCRIPTION
**Pull request by elikoga**
Structure changed, documentation did not

Also includes 0755 -> 0o755

### CODE REVIEW

File: doc/source/components/services.txt [hunk 1]
```diff
@@ -32,7 +32,7 @@ Alternatively to using Supervisor you can register each program as a system-wide
     class Tick(Component):
 
         def configure(self):
-            self += File('tick.sh', mode=0755)
+            self += File('tick.sh', mode=0o755)
```
Comment (ctheune) [inline doc/source/components/services.txt:35]: This could also be "rwxr-xr-x" today.

File: doc/source/user/quickstart.txt [hunk 2]
```diff
@@ -50,7 +50,7 @@ simple application:
       def configure(self):
           self += File(
               'tick.sh',
-              mode=0755,
+              mode=0o755,
```
Comment (ctheune) [inline doc/source/user/quickstart.txt:53]: see above

File: doc/source/user/quickstart.txt [hunk 3]
```diff
@@ -390,7 +394,7 @@ class:
...
```
Comment (ctheune) [inline doc/source/user/quickstart.txt:397]: see above

**Review by ctheune — CHANGES_REQUESTED** [review]
Thanks, great for catching that!

Would you mind also updating the octal modes to the newer "human readable" spellings? I marked a couple of occurrences, but not all.

File: doc/source/components/services.txt (hunk 1 above)
Comment (elikoga) [reply doc/source/components/services.txt:35]: "rwxr-xr-x" as a comment? Or 0o755, I'd rather the reader be aware, that both work

**Review by ctheune — APPROVED** [review]
Thanks!

**Pull request merged**